backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -351,7 +351,7 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
         |(
         |# add a .file in every empty directory to facilitate directory delocalization on the cloud
         |cd $cwd
-        |find . -type d -empty -print | xargs -I % touch %/.file
+        |find . -type d -empty -print0 | xargs --null -I % touch %/.file
```

Comment:
The `--null` version of the "Input file names are terminated by a null character" option is a GNU nicety that isn't supported by some versions of `xargs` like that in `alpine:latest`. The `-0` equivalent appears to be more portable.